src/ResourceManager/Resources/Commands.Resources/RoleAssignments/ValidateScope.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.WindowsAzure.Commands.Common
+{
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field)]
+    public class ValidateScope : ValidateArgumentsAttribute
+    {
+        protected override void Validate(object arguments, EngineIntrinsics engineIntrinsics)
+        {
+            if (arguments != null)
+            {
+                string scope = (string)arguments;
+                var pairs = scope.Split('/');
```

Comment:
sync'ed with Surabhi, so far we only want to check scopes that start with subscriptions.

---
In reply to: [118322183](https://github.com/Azure/azure-powershell/pull/4019#discussion_r118322183) [](ancestors = 118322183,118140390)